store/tikv/region_cache.go

```diff
@@ -208,6 +214,16 @@ func (r *Region) init(c *RegionCache) error {
 		rs.stores = append(rs.stores, store)
 		rs.storeEpochs = append(rs.storeEpochs, atomic.LoadUint32(&store.epoch))
 	}
+	// TODO(youjiali1995): It's possible the region info in PD is stale for now but it can recover.
+	// Maybe we need backoff here.
+	if len(availablePeers) == 0 {
+		return errors.Errorf("no available peers, region: {%v}", r.meta)
+	}
+	for i := len(availablePeers); i < len(r.meta.Peers); i++ {
+		r.meta.Peers[i] = nil
+	}
```

Comment:
What is the purpose of this code? I think gc should just work...

Reply:
GC can't work. https://github.com/pingcap/tidb/blob/3e41d1473f540a09ea96c8adb7193fff25ccbd7c/store/tikv/client_batch.go#L128-L135

I just copied it from https://github.com/pingcap/tidb/blob/3e41d1473f540a09ea96c8adb7193fff25ccbd7c/store/tikv/region_cache.go#L1220-L1222 but I think it's unnecessary too...

Reply:
I will remove it for clearness.